test/distributed/test_functional_api.py

```diff
@@ -252,6 +252,16 @@ def test_all_gather_into_tensor_coalesced(self, device):
         self.assertEqual(torch.ones([4 * dist.get_world_size()], device=device), res[0])
         self.assertEqual(torch.ones([4 * dist.get_world_size()], device=device) + 1, res[1])
 
+    @skip_if_lt_x_gpu(1)
+    def test_reduce_scatter_into_tensor_coalesced_cuda(self):
+        tensors = [torch.ones([4], dtype=torch.int64, device="cuda:0"), torch.ones([4], dtype=torch.int64, device="cuda:0") + 1]
```

Comment:
nit: device should be "cuda:rank" instead of 0? if we have more than 2 GPUs on the CI this would crash